src/main/java/List.java

```diff
@@ -0,0 +1,81 @@
+import java.lang.String;
+
+public class List {
+
+    public static String[] list = new String[100];
+    public static String[] done = new String[100];
+    public static int count = 0;
+    public static int doneCount = 0;
+
+    public static void printTask() {
+        System.out.println("    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n     "
+                + "Here are the tasks in your list:");
+        for (int i = 0; i < count; i++) {
+            System.out.println("     " + list[i]);
```

Comment:
You could consider defining some variable to deal with the huge block of spaces inside your `println`.